### PR TITLE
Support AIX operating system

### PR DESCRIPTION
**Pull request by ecnelises**

### CODE REVIEW

File: src/backend/libc/fs/types.rs [hunk 1]
```diff
@@ -798,7 +806,7 @@ bitflags! {
         const NOATIME = libc::ST_NOATIME as u64;
 
         /// `ST_NODEV`
-        #[cfg(any(target_os = "android", target_os = "emscripten", target_os = "fuchsia", target_os = "linux"))]
+        #[cfg(any(target_os = "android", target_os = "emscripten", target_os = "fuchsia", target_os = "linux", target_os = "aix"))]
```
Comment (sunfishcode) [inline src/backend/libc/fs/types.rs:809]: Please put the `target_os = "aix"` at the beginning of the `any` list, to keep it in alphabetical order.

**Review by sunfishcode — COMMENTED** [review]
Looks good; just one minor nit below, and rustfmt :-).

**Comment by sunfishcode** [comment]
I took the liberty of rebasing this, fixing the minor formatting issues, and marking it as ready for review. I don't have any way to test aix support, but if there are any further issues on aix, I'd be happy to accept further PRs.

**Comment by ecnelises** [comment]
Thanks!

**Pull request merged**